soh/soh/Enhancements/randomizer/logic.cpp

```diff
@@ -124,37 +280,62 @@ namespace Rando {
                 return IsChild;// || StickAsAdult;
             case RG_DEKU_SHIELD:
                 return IsChild;// || DekuShieldAsAdult;
+            case RG_WEIRD_EGG:
+                return IsChild;
+            case RG_RUTOS_LETTER:
+                return IsChild;
+
+                // Adult Trade
+            case RG_POCKET_EGG:
+                return IsAdult;
+            case RG_COJIRO:
+                return IsAdult;
+            case RG_ODD_MUSHROOM:
+                return IsAdult;
+            case RG_ODD_POTION:
+                return IsAdult;
+            case RG_POACHERS_SAW:
+                return IsAdult;
+            case RG_BROKEN_SWORD:
+                return IsAdult;
+            case RG_PRESCRIPTION:
+                return IsAdult;
+            case RG_EYEBALL_FROG:
+                return IsAdult;
+            case RG_EYEDROPS:
+                return IsAdult;
+            case RG_CLAIM_CHECK:
+                return IsAdult;
```

Comment:
I left these separate because I don't know what the implications are for glitched play. Pretty sure at least a couple of them can be turned in as child with glitches.